abtesting/Tests_iOS/Tests_iOS.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  Copyright 2021 Google LLC
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import XCTest
+
+class Tests_iOS: XCTestCase {
+  override func setUpWithError() throws {
+    // Put setup code here. This method is called before the invocation of each test method in
+    // the class.
+
+    // In UI tests it is usually best to stop immediately when a failure occurs.
+    continueAfterFailure = false
+
+    // In UI tests it’s important to set the initial state - such as interface orientation -
+    // required for your tests before they run. The setUp method is a good place to do this.
+  }
+
+  override func tearDownWithError() throws {
+    // Put teardown code here. This method is called after the invocation of each test method in
+    // the class.
+  }
+
+  func testExample() throws {
```

Comment:
It would be great to add real tests here, but that can be a subsequent PR.

Reply:
Yes, I can make that a subsequent PR 👍